### DISCUSSION TITLE
Coping para comportamentos antissociais entre profissionais de T.I.

### DISCUSSION THREAD

**Discussion by fititnt** [frontendbr/forum#530] [Dicas]
Repositório: https://github.com/fititnt/coping-comportamento-antissocial-ti
TL;DR: **Estudo aplicado de mecanismos de enfrentamento (coping) para desestimular de forma educativa comportamentos antissociais em interações entre profissionais de Tecnologia da Informação. Lista de sugestões de como reagir quando colega agir de forma antissocial com os demais**

---
[Estratégias de coping, ou enfrentamento](https://pt.wikipedia.org/wiki/Mecanismos_de_enfrentamento) são esforços cognitivos e comportamentais para lidar com situações de dano, de ameaça ou de desafio quando não está disponível uma rotina ou uma resposta automática.  Ter [comportamento antissocial](https://pt.wikipedia.org/wiki/Comportamento_antissocial), em algum momento **não** necessariamente implica em ter [transtorno de personalidade antissocial](https://pt.wikipedia.org/wiki/Transtorno_de_personalidade_antissocial), com frequência é **[resultado principalmente da cópia comportamento observado de outros](https://www.facebook.com/fititnt/posts/1408436005874543)** e, diferente da [misantropia](https://pt.wikipedia.org/wiki/Misantropia), está relacionada a ignorar, **mesmo que por falta de conhecimento**, a possibilidade de estar afetando negativamente outras pessoas, por falta de [empatia](https://pt.wikipedia.org/wiki/Empatia) com o  sofrimento de outras.

Este projeto é restrito a interações sociais entre **profissionais de Tecnologia da Informação**. Assume **premissas como a de que remover a percepção de padrões duplos** e de que **sugerir comportamentos que não apenas minimizem problemas, mas também NÃO cause prejuízo pessoal ao replicador**, é mais eficiente para re-educar e tornar qualquer pessoa que observa um potencial replicador.

---

Existem vários comportamentos antissociais. Alguns são mais fáceis de reclamar sem receber críticas, como [o caso de **xenofobia** em novembro e dezembro de 2016](https://github.com/APDA-HQ/APDA/issues/43) e outros casos de **racismo**. Eu sei que muitos aqui diriam que isso é excessão, e costuma ser, mas em especial no Facebook tem grupos que fazem bully com colegas. E esses casos são bem mais complicados, tomam tempo desnecessários em discussões que costumam não ser construtivas.

Um dos objetivos desse repositório é desincentivar críticas pouco relevantes ao incentivar **abordagens construtivas com pessoas que estão aprendendo**. Em alguns grupos é comum até guerras imaginárias de linguagem vs linguagem, induzir/forçar/ridicularizar pessoas que estão aprendendo a desistir de algo para usar outra coisa extremamente complexa que ela não precisa, e no fim pessoas podem deixar de contribuir com código aberto por medo de críticas.

**Comment by cristofersousa** [comment]
Tu subiu essa issue em 2017, 03 anos depois veio a pandemia, falamos tanto sobre saúde mental e o tema Coping passou despercebido.  No caso eu não conhecia o termo até assistir a palestra da minha psicologa no caso ela não abordou como enfrentamento social mas sim como lidar com stress rotineiro.

Talvez valha a pena reviver essa thread, dado que estamos mais do que nunca em um momento cáotico.

Link da Apresentação: [https://youtu.be/sPA8x70HtiQ?t=6523](https://youtu.be/sPA8x70HtiQ?t=6523)

**Comment by cristofersousa** [comment]
Sobre mecanismos de defesas:

O conceito de mecanismo de defesa foi originalmente sugerido por Sigmund Freud, que argumentou que as reações defensivas ocorrem quando o ego tenta se proteger, na psicologia contemporânea, o termo mecanismo de defesa é usado de forma mais ampla para se referir a qualquer padrão de comportamento que as pessoas usam para se proteger de emoções desagradáveis, como vergonha, raiva e culpa.

> TIPOS DE MECANISMOS DE DEFESA

Qualquer comportamento destinado a se proteger de emoções desagradáveis ​​pode ser um mecanismo de defesa. No entanto, alguns mecanismos de defesa são tão comuns que têm sido bastante citados em textos psicológicos. Esses incluem:

> Projeção  

O ato de projetar os próprios sentimentos inconscientes sobre o outro. Por exemplo, uma esposa pode insistir que seu marido parece zangado quando na verdade é ela quem está com raiva.

> Negação 

Recusar-se a reconhecer uma verdade ou emoção desagradável. A negação é amplamente citada como a primeira etapa do processo de luto após uma perda significativa.
 
> Somatização 

Transferência de sentimentos negativos em sintomas físicos. Por exemplo, um homem pode desenvolver problemas estomacais toda vez que fica ansioso.

> Formação de Reação

Atuando exatamente o oposto de seus desejos ou pensamentos inconscientes. Por exemplo, um homem que é um cristão devoto que se sente sexualmente atraído por outros homens pode se tornar extremamente homofóbico porque a homossexualidade não é aceita em sua cultura religiosa.

George Eman Vaillant classificou os mecanismos de defesa em níveis de desenvolvimento psicológico. Seus níveis incluem:

- Patológico
- Imaturo
- Neurótico
- Maduro

De acordo com Vaillant, esses níveis se correlacionam aproximadamente com os estágios do desenvolvimento psicológico humano, e certos mecanismos de defesa são mais propensos a se desenvolver em certas idades nessa concepção. Mecanismos de defesa maduros fazem parte de relacionamentos adultos saudáveis ​​e incluem comportamentos como empatia, humor, altruísmo e identificação.

1. [https://www.goodtherapy.org/blog/psychpedia/coping-mechanisms](https://www.goodtherapy.org/blog/psychpedia/coping-mechanisms)

![image](https://user-images.githubusercontent.com/4699770/177461939-5c4b4d37-34af-4b48-a0ea-90bb5100acf1.png)
